.github/workflows/ci-spell-checking.yml

```diff
@@ -0,0 +1,44 @@
+name: ci-spell-checking
+
+on:
+  push:
+    branches: 
+    - main
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+    branches: 
+    - main
+
+jobs:
+  lint:

```

Comment:
this should be renamed to something meaningful ... and should not be `lint`.

Reply:
Do you have any suggestion?